etc/config/c.amazon.properties

```diff
@@ -1247,76 +1339,63 @@ compiler.carmgtrunk.name=ARM gcc trunk (linux)
 compiler.carmgtrunk.semver=trunk
 
 # 64 bit
-group.cgcc64arm.groupName=Arm 64-bit GCC
-group.cgcc64arm.compilers=caarchg54:carm64g630:carm64g640:carm64g730:carm64g750:carm64g820:carm64g850:carm64g930:carm64g940:carm64g950:carm64g1020:carm64g1030:carm64g1040:carm64g1100:carm64g1120:carm64g1130:carm64g1210:carm64gtrunk:carm64g1220
+group.cgcc64arm.groupName=ARM64 gcc
+group.cgcc64arm.compilers=caarchg54:carm64g630:carm64g640:carm64g730:carm64g750:carm64g820:carm64g850:carm64g930:carm64g940:carm64g950:carm64g1020:carm64g1030:carm64g1040:carm64g1100:carm64g1120:carm64g1130:carm64g1210:carm64gtrunk:carm64g1220:carm64g1310
 group.cgcc64arm.isSemVer=true
 group.cgcc64arm.objdumper=/opt/compiler-explorer/arm64/gcc-10.2.0/aarch64-unknown-linux-gnu/aarch64-unknown-linux-gnu/bin/objdump
 group.cgcc64arm.instructionSet=aarch64
 
 compiler.caarchg54.exe=/opt/compiler-explorer/arm64/gcc-5.4.0/aarch64-unknown-linux-gnueabi/bin/aarch64-unknown-linux-gnueabi-gcc
-compiler.caarchg54.name=ARM64 gcc 5.4
```

Comment:
Hi, thanks for adding these!

Hmm I think this change caused https://github.com/compiler-explorer/compiler-explorer/issues/5040 I assume the idea is that somewhere there's a missing piece of code that's supposed to add the version after the group name?

Reply:
Argl, yes